pkg/services/sqlstore/migrations/dashboard_mig.go

```diff
@@ -234,4 +234,7 @@ func addDashboardMigration(mg *Migrator) {
 	mg.AddMigration("Add isPublic for dashboard", NewAddColumnMigration(dashboardV2, &Column{
 		Name: "is_public", Type: DB_Bool, Nullable: false, Default: "0",
 	}))
+
+	mg.AddMigration("Remove isPublic for dashboard", NewRawSQLMigration(
+		"ALTER TABLE dashboard DROP COLUMN is_public"))
```

Comment:
not sure have dropped a column before. @bergquist is this safe? This could make downgrading impossible, user upgrades to 9.1 removes this column, then due to issue is downgraded, the column is no longer there (but is included in SQL selects) causing errors.